src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/NormalizationExceptionTelemetryProcessor.cs

```diff
@@ -7,15 +7,20 @@
 using System.Linq;
 using EnsureThat;
 using Microsoft.Health.Common.Telemetry;
+using Microsoft.Health.Fhir.Ingest.Service;
 using Microsoft.Health.Fhir.Ingest.Template;
-using Microsoft.Health.Logging.Telemetry;
-
+using Microsoft.Health.Logging.Telemetry;
+
 namespace Microsoft.Health.Fhir.Ingest.Telemetry
 {
     public class NormalizationExceptionTelemetryProcessor : ExceptionTelemetryProcessor
     {
         private readonly string _connectorStage = ConnectorOperation.Normalization;
-        private static readonly Type[] DefaultExceptions = new[] { typeof(IncompatibleDataException) };
+        private static readonly Type[] DefaultExceptions = new[]
+        {
+            typeof(IncompatibleDataException),
+            typeof(InvalidDataFormatException),
```

Comment:
Adding it as a handled exception so we don't stall on events that are on unsupported formats, but let me know if this is not desired.